cumulusci/cli/cci.py

```diff
@@ -1673,7 +1677,10 @@ def _get_click_options_for_task(self, task_options):
             # task level via task._validate_options() or Pydantic models.
             click_options.append(
                 click.Option(
-                    param_decls=(f"--{name}",),
+                    param_decls=(
+                        f"--{name}",
+                        f"--{name.replace('_', '-')}",
```

Comment:
This leads to showing even options that don't have an underscore twice in --help:

     --namespace, --namespace TEXT   Package namespace

Let's add the normalized form only if it's different.

Reply:
09a046b